Production_scripts/B05a_Process_A&E_(AE2)_extract.R

```diff
@@ -0,0 +1,1094 @@
+#####################################################
+# A&E Extract
+# Author: Catherine Holland
+# Date: February 2022
+# Written on RStudio Server
+# Version of R - 3.6.1
+# Input -
+# Description - Process A & E extract
+#####################################################
+
+
+library(dplyr)
+library(tidyr)
+library(ggplot2)
+library(createslf)
+library(slfhelper)
+
+
+## get data ##
+
+# latest year
+latest_year <- 1920
+
+# function here till PR pushed
+get_year_dir <- function(year, extracts_dir = FALSE) {
+  year_dir <- fs::path("/conf/sourcedev/Source_Linkage_File_Updates", year)
+
+  year_extracts_dir <- fs::path(year_dir, "Extracts")
+
+  return(dplyr::if_else(extracts_dir, year_extracts_dir, year_dir))
+}
+
+
+ae_episode_extract <- readr::read_csv(
+  paste0(
+    get_year_dir(year = latest_year),
+    "/Extracts/A&E-episode-level-extract-20",
+    latest_year,
+    ".csv.gz"
+  )
+) %>%
```

Comment:
I think you mean get_boxi_extract which has now been pushed. We are tidying up PR's so i'll make this change :)